### ISSUE TITLE
Tweaking recipes for creative items, or power generation

### ISSUE THREAD

**Issue by BrotherGluttony** [al132mc/antimatter-chemistry#307]
Enjoying this pack SO much but:

TLDR: The sheer amount of ore you need for creative item singularities is ridiciculous and without the rainbow gen you cant power enough enchanted quarries without MASSIVE fusion reactors that are killer on server health, please tweak something to make it less of a pain to get creative items <3

 ive been playing for months and the amount of power you need to consume to power enough enchanted quarries to ore gen enough items to make ALL of the singulatiries you need for creative items is ridiculous, i have 3 torrid 5 fusion reactors chaining difffernt fuel types and only gen 700k rf/t which has kept my 2 quarries going for at least 2 months of playing daily with offline chunkloaders and im not even close to getting half the singularities. Even now im just getting enough mats to try and automate the Rainbow Generator, but the gap between late game and creative is entirely too large for this small of a pack.

**Comment by MasterEnderman** [comment]
I guess some points are related to #217 as I already mentioned some changes there.
Furthermore I slightly decreased the base energy consumption of the quantum quarry.
Besides the fusion reactors you could also try to build a molten salt reactor / turbine combination to power your quarries.

**Comment by ObsidianWolff** [comment]
i was gonna go that route but there wasnt much info i could find on building them aside from the old nuclearcraft video, are they more efficient than other reactors? if they are a level up in  terms of power gen id def be down to figure it out and potentially share updated guides on them :D
